packages/create-app/templates/template-next-static-export/README.md.hbs

```diff
@@ -30,4 +30,6 @@ Production configuration is stored in `.env.production`.
 
 If you did not fill in the URL your production application will be hosted on you will need to fill in the `NEXT_PUBLIC_FOUNDRY_REDIRECT_URL` in `.env.production`.
 
-In order to make API requests to Foundry, CORS must be configured for the stack to allow the production origin to load resources. The configured OAuth client must also allow the production origin auth callback as a redirect URL.
+In order to make API requests to Foundry, CORS must be configured for the stack to allow the production origin to load resources. This will be automatically done for you if you are using Foundry website hosting. The configured OAuth client must also allow the production origin auth callback as a redirect URL.
+
+A `foundry.config.json` file is included in the root of this project to make deploying to Foundry website hosting with [`@osdk/cli`](https://www.npmjs.com/package/@osdk/cli) easier. If you are not using Foundry website hosting for your application you may delete this file.
```

Comment:
What do you think we make the generation of the file opt-out option with a default value of true. And here we can mention, If you aren't using foundry website hosting pass `--websiteHosting false`. I feel asking the user to delete here is a bit unclean. What do you think?